doc/installation-guide.md

```diff
@@ -0,0 +1,61 @@
+Step by step tutorial to install FastMoE on your local machine:
+
+1. First of all you'll need to check your torch and nccl version, make sure to have a CUDA version compatible to the one torch was compiled (in general if you have the latest torch version it works also with the latest CUDA):
+```
+# go in terminal and use this command, the output should be something like this:
+
+python -c  'import torch; print(torch.__version__); print(torch.cuda.nccl.version())'
+>>> 2.0.1+cu117
+>>> (2, 14, 3)  # -> this means version 2.14.3
+
+# to check cuda version just type to see which cuda version you are using, should output something like this:
+
+which nvcc
```

Comment:
Use ` nvcc --version` instead of checking its path. (maybe add that `which nvcc` may also help)